pycds/orm/tables.py

```diff
@@ -53,11 +59,17 @@ class Network(Base):
     contact_id = Column(Integer, ForeignKey("meta_contact.contact_id"))
 
     stations = relationship(
-        "Station", backref=backref("meta_network", order_by=id)
+        "Station", order_by="Station.id", back_populates="network"
     )
-    variables = relationship(
-        "Variable", backref=backref("meta_network", order_by=id)
+    meta_station = synonym("stations")
+    variables = relationship("Variable", back_populates="network")
+    meta_vars = synonym("variables")
+    contact = relationship("Contact", back_populates="networks")
```

Comment:
`networks`? Aren't `Network` and `Contact` in a 1-to-1 relationship?

Reply:
In practice, this relationship might always be 1:1, but the original design establishes a 1:N relationship between Contact and Network, and in principle the same Contact could serve for many Networks. The plural is used because SQLAlchemy will always return a *list* of Networks for this attribute.